back-end/e-learning/app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+  function login(Request $request)
+  {
+    $user= User::where('email', $request->email)->first();
+    if (!$user || !Hash::check($request->password, $user->password)) {
+      return response([
+        'message' => ['These credentials do not match our records.']
+      ], 404);
+    }
+
+    $token = $user->createToken($user->email.'my-app-token')->plainTextToken;    
+            
+		$response = [
+      'user' => $user,
+      'token' => $token
+    ];
+        
+  	return response($response, 201);
+  }
+
+  public function me(){
+  	$user = User::find(auth()->user()->id);
+    return response($user, 201);
```

Comment:
Is it not possible to just return `auth()->user()` as response, since it just returns the `User` object of the Authenticated User. Please confirm.

Reply:
yes ma'am, that's possible. I'll update that one. Thank you.